.github/workflows/rust.yml

```diff
@@ -244,7 +247,9 @@ jobs:
 
       - name: Build tests
         if: matrix.directory == 'packages/engine'
-        run: cargo test --manifest-path ${{ matrix.directory }}/Cargo.toml --no-run ${{ matrix.flags }}
+        run: |
```

Comment:
Missed comment on this one

Reply:
Oh, I forgot I added this as well. Previously I only had to update two locations, thanks: ad827f983ef1f61fb1ac1cfb9a109b7f35d9e285